posthog/api/decide.py

```diff
@@ -141,7 +141,10 @@ def get_decide(request: HttpRequest):
                     ),
                 )
 
-            property_overrides = get_geoip_properties(get_ip_address(request))
+            property_overrides = {}
+            if (data.get("geoip_disable") or False) is False:
```

Comment:
unsure how to best test this 🤔

Reply:
make a decide request with `geoip_disable` in there, add a feature flag that depends on the country code, add an ip from a different country, see it pass/fail based on whether this setting is on or off

Reply:
the decide code + geoip property overrides confuses me

Reply:
am i supposed to check based on the ip I pass in to the post decide call, whether that returns me the ip based feature flags or not?

Reply:
> am i supposed to check based on the ip I pass in to the post decide call, whether that returns me the ip based feature flags or not?

Yes, say if the flag depends on country = Australia, and the ip is australia, then it should return `False` if geoip_disable is set, and no person properties exist.

Reply:
In the decide response, do you mean empty flags? What is `False` here?

Reply:
use decide v3, and in that case, yes decide returns `{'beta-feature': False}`,

Reply:
i'm just getting empty flags : ( do you want to huddle?

Reply:
Note: we found the issue! It's the data arg in _post_decide is confusing because data completely overrides the default parameters which include values that are needed for decide to evaluate as expected